tools/install-bin-windows.bat

```diff
@@ -15,8 +15,23 @@ if not defined TINYTEX_VERSION (
   set TINYTEX_URL=https://github.com/yihui/tinytex-releases/releases/download/v%TINYTEX_VERSION%/%TINYTEX_INSTALLER%-v%TINYTEX_VERSION%.zip
 )
 
-rem download the zip package and unzip it
+rem download the zip package - method 1
+echo Download zip file... Method 1
+powershell -Command "& { try {Add-Type -A 'System.Net.Http'; [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12;$response = (New-Object System.Net.Http.HttpClient).GetAsync($Env:TINYTEX_URL); $response.Wait(); $outputFileStream = [System.IO.FileStream]::new('install.zip', [System.IO.FileMode]::Create, [System.IO.FileAccess]::Write); $response.Result.Content.CopyToAsync($outputFileStream).Wait(); $outputFileStream.Close()} catch {throw $_}}"
+if ,not errorlevel 1 goto unzip
```

Comment:
What does the comma do here?

Reply:
😞  nothing. It is an error. I commited to fast. thanks for the review!